code/modules/mob/living/simple_mob/subtypes/mechanical/disbot_vr.dm

```diff
@@ -0,0 +1,70 @@
+
+/datum/category_item/catalogue/technology/drone/infectionbot
+	name = "Drone - Injection Drone"
+	desc = "A strange and aged drone, this model appears to be made for gathering of genetic samples,\
+	sacrificing both power and durability for storage space and advanced scanners.\
+	The drone has clear dents and scratches all over it's casing bulging inwards in multiple areas,\
+	some even penetrated showing the advanced and headache inducing parts inside.\
+	<br><br>\
+	This model in particular has a collection of hostile and spinetingeling parts on the underside,\
+	small advanced anti-gravity generators located between giant syringes made for injecting and extracting all manners of fluids\
+	and samples of soil or other biological matter.\
+	Most interesting is the syringe it uses after taking a sample of any creature, the dry, flakey substance\
+	it injects has traces of bicardine and innaprovaline, now no more than rotten and expired toxins however.\

```

Comment:
...Based on this description, I think it would be more lore fitting that, instead of zombie powder, the injected toxin is expired_medicine instead.

Reply:
Isn't debilitating at all and makes the mob moat. I'm personally not going to do so.

Reply:
Expired_medicine deals iirc 5 toxin per unit injected  5u injected per hit means 25 damage.

You can ignore it, but the thing about toxin is that if its left untreated and its around 50/60+, it will begin damaging your liver. Once your liver is dead, it moves on to other organs. It's either you fix it or you die. 

Did you test to see whether or not the mob continues to attack you while youre downed under the effects of zombie powder?

Reply:
If that is the case, than the mob's lore should be rewritten to compensate. Lore should fit what the mob is. It makes no sense that a super deadly toxin is coming from a mob that is... otherwise... fairly mundane in both catalog reward and description.

Reply:
I doubt expired medicine is "dry, flakey"

Reply:
It can be, if it's expired enough.

Reply:
As I said, not against the idea of an extremely dangerous mob, but, description should match the stats. If it is a murderdeathbot, it should be described as such. It is a bit of a stretch for what is essentially a malfunctioning probe to be so dangerous, especially since Zombie Powder shares pretty much none of the reagents of Inaprovaline or Bicaridine. There is no reason why those expired medicines would suddenly synthesize into Zombie Powder.

Reply:
done